src/main/java/com/meilisearch/sdk/model/Key.java

```diff
@@ -13,29 +13,25 @@
  * @see <a href="https://www.meilisearch.com/docs/reference/api/keys">API specification</a>
  */
 @Getter
+@Setter
```

Comment:
I understand the move, but now the user can redefine the uid/key, which they are not allowed to (directly). Do you think is there any other way to handle that?

Reply:
Yeah, the way to do this would be to use runtime reflection, I think. I wrote a bit about this in my third bullet in my PR description. If you want me to do this, let me know.

Reply:
It will not be a huge deal in any case. No need to change!